test/Microsoft.Buld.Sql.Tests/DotnetTestBase.cs

```diff
@@ -294,5 +294,18 @@ protected void VerifyDacPackage(bool expectPreDeployScript = false, bool expectP
                 }
             }
         }
+
+        /// <summary>
+        /// Verifies the project file has the expected target platform
+        /// </summary>
+        /// <param name="expectedTargetPlatform">The expected target platform</param>
+        protected void VerifyTargetPlatform(string expectedTargetPlatform)
```

Comment:
nit: I think this will only be used by the `dotnet new` template tests, I'm fine with it being a private function there

Reply:
leaving as a protected function for now since it's being used by the derived class TemplateTests, private would make it only accessible within the DotnetTest class
(I may be missing something and this can certainly be changed in the future)

Reply:
Sorry I meant to say a private function in the TemplateTests class

Reply:
ah, that makes sense now. will keep that in mind for the future - thanks!